README.md

```diff
@@ -7,12 +7,19 @@ Drupal 8 theme based on the [Europa Component Library][1] (ECL).
 **Table of contents:**
 
 - [Installation](#installation)
+- [Requirements](#requirements)
 - [Development](#development)
   - [Project setup](#project-setup)
   - [Using Docker Compose](#using-docker-compose)
   - [Disable Drupal 8 caching](#disable-drupal-8-caching)
   - [Working with ECL components](#working-with-ecl-components)
-- [Demo module](#demo-module)
+- [Demo module](https://github.com/openeuropa/oe_theme/blob/master/modules/oe_theme_demo/README.md)
```

Comment:
Shouldn't we link the local file here? Like:

```
[Demo module](./modules/oe_theme_demo/README.md)
```

Then it will always point to the right file regardless on the branch.